FrejaEidClient/src/main/java/com/verisec/frejaeid/client/client/impl/AuthenticationClient.java

```diff
@@ -72,6 +76,17 @@ public String initiate(InitiateAuthenticationRequest initiateAuthenticationReque
         return reference;
     }
 
+    @Override
+    public byte[] initiateQRCodeAuthentication(String reference) throws FrejaEidClientInternalException, FrejaEidException, IOException {
+        LOG.debug("Initiating generation of QR code for authentication from transaction reference {}.", reference);
+        String encodedReference = URLEncoder.encode(reference, StandardCharsets.UTF_8.toString());
+        String bindTransactionToUserUrl = "frejaeid://bindUserToTransaction?transactionReference=" + encodedReference;
```

Comment:
I would extract prefix "frejaeid://..." to a constant.

Reply:
Done.